Assets/Scripts/BulletManagement/PlayerEmitter.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
+
+namespace BulletManagement
+{
+    public class PlayerEmitter : EmitterBase
+    {
+        private void Start()
+        {
+            this.bulletPattern = this.GetComponent<BulletPattern>();
+        }
+        
+        // Update is called once per frame
+        private void Update()
+        {
+            timeCounter += Time.deltaTime * bulletSpeedMultiplier;
+            if (Input.GetButton("Fire1"))
+            {
+                if (timeCounter >= emitFrequency)
+                {
+                    EmitBullets();
+                    timeCounter = 0f;
+                }
+                // EmitBullets();
```

Comment:
Is there a reason for the commented out function here?

Reply:
Artifact of testing things. Removed.